uint8  id
string state
uint8  task_state
